src/components/Tabs.js

```diff
@@ -22,8 +22,8 @@ const TabsFactory = ({ type = 'standard', ...props }) => {
   );
 };
 
-TabsFactory.propTypes = Object.assign({
-  type: PropTypes.oneOf(Object.keys(TabsTypes))
-}, Tabbable.PropTypes);
```

Comment:
I don't think you need to do this conversion and I also don't think this is correct. Before we were merging one object into another:
```
Object.assign({type: ....}, Tabble.PropTypes)
```
Now you are flattening one object an array that is a property on an object.
```
{ type: PropTypes.oneOf([..., ...Tabable.PropTypes])}
```